backend/app.py

```
@@ -158,7 +159,8 @@ def add_item():
         return jsonify({"message": "Item added successfully", "item": item})
 
     except Exception as e:
-        return jsonify({"error": str(e)}), 500
+        logging.error("Error in add_item: %s", traceback.format_exc())
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)